MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift

```diff
@@ -49,7 +57,7 @@ extension MovieDetailViewController: DesignProtocol {
     func styleViews() {
         view.backgroundColor = .white
         
-        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: 1000)
+        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: 1400)
```

Comment:
This shouldn't be set manually. I've commented previously how adding top, bottom and width constraints eliminate the need to force scrollview content size.

Reply:
Changed in PR #35